config/postgrest.conf

```diff
@@ -0,0 +1,26 @@
+# postgrest.conf
```

Comment:
I don't think this file is doing anything (without a bind mount)?

Reply:
removed.